internal/daemonset/daemonset.go

```diff
@@ -310,3 +322,44 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 
 	return labels
 }
+
+func MakeLoadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	loadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
```

Comment:
Why do we need to check that `len(ra.Load)>0` if it is required in the spec that `minItems=1`?

Reply:
We might have only specified `unload` arguments and let `load` empty. That is valid from the CRD's perspective: `minItems=1` means "at least one item when the list is not nil".